packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -152,7 +113,19 @@ export function CardContainer(props: CardContainerProps) {
       )
     }
     const listIds = dataRows.map(el => el.values[userIdColumnIndex])
-    cards = <UserCardList data={data} list={listIds} size={MEDIUM_USER_CARD} />
+    cards = (
+      <UserCardList
+        data={{
+          ...queryMetadata,
+          queryResult: {
+            queryResults: rowSet!,
+            concreteType: 'org.sagebionetworks.repo.model.table.QueryResult',
+          },
```

Comment:
`UserCardList` expects the full QueryResultBundle, so reconstruct it. Given infinite time, I would have refactored this. This should not be a typical pattern.

Reply:
Recommend adding a loud comment, like
// DO NOT COPY!